quickstart_guides/ruby.md

```diff
@@ -2,3 +2,84 @@ address: support.aptible.com/quickstart/ruby
 
 ### Ruby Quickstart
 
+This guide assumes you have:   
+- An Aptible account,  
+- An SSH key associated with your Aptible user account, and  
+- The Aptible command line tool installed
+
+#### 1. Provision Your App  
+Tell the Aptible API that you want to provision an application. Until you push code and trigger a build, Aptible uses this as a placeholder.
+
+##### Command line
+Use the `apps:create` command: `aptible apps:create HANDLE`
+
+For example: 
+```
+aptible apps:create my-example-app
+```
+
+If you have multiple accounts, you will need to specify `--account <account_handle>`. Each account handle is listed next to the account name in the Aptible Dashboard.
```

Comment:
This won't be the case for first-time user experience (FTUX) right now, so I'd omit it to keep this simpler.